demo.html

```diff
@@ -4,11 +4,93 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+    <title>Movie</title>
+    <link rel="stylesheet" href="demo.css">
+    <link rel ="icon" type="image/x-icon" href="/hello/favicon.ico">
```

Comment:
change to relative path (create hello folder inside repo and add favicon inside that and change `href="./hello/favicon.ico"` or `href="hello/favicon.ico"`